packages/firestore/CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # Unreleased
+- [feature] Added Node.JS support for Cloud Firestore.
```

Comment:
It's spelled "Node.js".

Also should we note that this support is limited (i.e. only with persistence disabled)?

Reply:
Good points. Fixed.